kubernetes/gke-base/docs/index.rst

```diff
@@ -0,0 +1,430 @@
+
+Overview
+--------
+
+This demo shows a deployment of |cp| on Google Kubernetes Engine (GKE) leveraging |co-long| with random data generation provided via the `Kafka Connect Datagen <https://www.confluent.io/hub/confluentinc/kafka-connect-datagen>`__.
+
+The major components of this demo are:
+
+* A Kubernetes cluster running on GKE.
+* |co-long| which is used to manage the following |cp| components
+
+  * A single node |zk|
+  * A single node |ak|
+  * A single node |sr|
+  * |c3|
+  * A single node |kconnect-long|
+  * One instance of `kafka-connect-datagen` to produce randomly generated data
+
+.. figure:: images/operator.png
+    :alt: operator
+
+Demo Preqrequisites
+-------------------
+The following applications or libraries are required to be installed and available in the system path in order to properly run the demo.
+
++------------------+----------------+---------------------------------------------------------+
+| Application      | Tested Version | Info                                                    |
++==================+================+=========================================================+
+| ``curl``         | ``7.54.0``     | https://curl.haxx.se/                                   |
++------------------+----------------+---------------------------------------------------------+
+| ``kubectl``      | ``1.14.3``     | https://kubernetes.io/docs/tasks/tools/install-kubectl/ |
++------------------+----------------+---------------------------------------------------------+
+| ``helm``         | ``2.14.3``     | https://helm.sh/docs/using_helm/#install-helm           |
++------------------+----------------+---------------------------------------------------------+
+| ``jq``           | ``1.6``        | https://stedolan.github.io/jq/                          |
++------------------+----------------+---------------------------------------------------------+
+| ``gcloud``       | ``259.0.0``    |  https://cloud.google.com/sdk/install                   |
+| ``GCP sdk core`` | ``2019.08.23`` |                                                         |
++------------------+----------------+---------------------------------------------------------+
+
+Running the Demo
+----------------
+
+.. warning:: This demo uses the real GCP SDK to launch real resources. To avoid unexpected charges, carefully evaluate the cost of resources before launching the demo and ensure all resources are destroyed after you are done evaluating the demonstration.  Optionally, refer to the |co| :ref:`Sizing Recommendations <co-env-sizing>` document and the :ref:`examples-operator-gke-base-variable-reference` section for more information on required resources for running |cp| on Kubernetes.
+
+ 
+Setup
+*****
+
+Clone the Confluent examples repository and change directories on your termianl into the ``gke-base`` directory.
+
+.. sourcecode:: bash
+
+		git clone git@github.com:confluentinc/examples.git; cd examples/operator/gke-base
```

Comment:
```suggestion
		git clone git@github.com:confluentinc/examples.git
		cd examples/operator/gke-base
```

Reply:
I kind of like it in one line for copy and paste purposes... thoughts?

Reply:
1. Docs now have a copy/paste button for easy copy.

2. Generally speaking, examples should err on the side of being more explicit to help users learn.

Reply:
I thought users might run into issues if multiple commands are in the same "code snippet" block, which is why I did this one this way.  I'm not certain any/all commands will have issues, so I don't have evidence this for sure is an issue.  I've definitely tried to keep command output out of the snippet blocks for this reason.   If you think it's save to "stack" commands like this I'll break it up

Reply:
Yeah, "stacking" commands is easier to grok, especially for new users.  But this is just a nit, do what you deem better.

Reply:
I tested them stacked and didn't see any issue.  If we see field issues I can adjust accordingly.